gradle.properties

```diff
@@ -13,12 +13,12 @@ loader_version_range=[47,)
 
 dynamictrees_version_range=[0,)
 target_mod_id=vinery
-target_mod_version_range=[1.4.25,)
+target_mod_version_range=[1.4.32,)
 
 mod_id=dtvinery
 mod_name=Dynamic Trees for Vinery
 mod_name_s=DynamicTreesforVinery
-mod_version=1.1
+mod_version=1.2
```

Comment:
A change in the second version number would require a major change in the mod content, which currently does not exist.

Reply:
I changed the version to 1.2 because it's only compatible with latest Vinery. Do you want me to change it to 1.1.1 ?